docker/dev/php/Dockerfile

```diff
@@ -2,43 +2,52 @@ FROM php:7.3-cli-alpine3.10
 
 # https://getcomposer.org/doc/03-cli.md#composer-allow-superuser
 ENV COMPOSER_ALLOW_SUPERUSER 1
-ARG HOST_USER
 ARG HOST_USER_ID
 
-COPY files/ini /phpIni
+COPY files/php /phpIni
+COPY files/bin/ /usr/local/bin/
 
 # SYS: Install required packages
 RUN apk --no-cache upgrade && \
-    apk --no-cache add bash git sudo nano openssh autoconf gcc g++ make gettext
+    apk --no-cache add bash git sudo openssh autoconf gcc g++ make gettext make shadow
 
-#sadly apple users run into a problem since the AD user has a higher ID than 65k :(
-#so we do it oldschool
-RUN echo "$HOST_USER:x:$HOST_USER_ID:82:Linux User,,,:/home/$HOST_USER:" >> /etc/passwd && \
-    echo "$HOST_USER:!:$(($(date +%s) / 60 / 60 / 24)):0:99999:7:::" >> /etc/shadow && \
-    echo "$HOST_USER:x:$HOST_USER_ID:" >> /etc/group && \
-    mkdir /home/$HOST_USER && \
-    chown $HOST_USER:$HOST_USER /home/$HOST_USER && \
-    echo "ALL ALL=NOPASSWD: ALL" >> /etc/sudoers && \
-    addgroup $HOST_USER www-data
+RUN apk add librdkafka librdkafka-dev \
+        --update-cache --repository http://dl-3.alpinelinux.org/alpine/edge/community
 
-COPY files/user-home /home/$HOST_USER
+# SYS: add ssh key (for GitHub)
+COPY --chown=www-data:www-data files/.ssh /home/www-data/.ssh
+
+RUN chown -R www-data:www-data /home/www-data && \
+    chmod 400 /home/www-data/.ssh/config
+
+# Set host user id if given, to avoid permission conflicts in dev
+RUN /bin/bash -c 'if [ -n "$HOST_USER_ID" ] && [ "$HOST_USER_ID" -lt 60000 ]; then \
+        usermod -u ${HOST_USER_ID} www-data; \
+    fi'
+
+COPY files/user-home /home/www-data
 
 # SYS: add ssh config (for GitHub)
 COPY files/.ssh/config /root/.ssh/config
-COPY files/.ssh/config /home/$HOST_USER/.ssh/config
+COPY files/.ssh/config /home/www-data/.ssh/config
 RUN ln -s /run/secrets/ssh_host_key /root/.ssh/id_rsa && \
-    ln -s /run/secrets/ssh_host_key /home/$HOST_USER/.ssh/id_rsa && \
-    chown -R $HOST_USER:www-data /home/$HOST_USER && \
+    ln -s /run/secrets/ssh_host_key /home/www-data/.ssh/id_rsa && \
+    chown -R www-data:www-data /home/www-data && \
     chown root:root /root/.ssh/config && \
-    chmod 700 /home/$HOST_USER/.ssh && \
-    chmod 400 /home/$HOST_USER/.ssh/config /root/.ssh/config
+    chmod 700 /home/www-data/.ssh && \
+    chmod 400 /home/www-data/.ssh/config /root/.ssh/config
+
+# PHP: Install php extensions
+RUN pecl channel-update pecl.php.net && \
+    pecl install pcov rdkafka && \
+    php-ext-enable pcov rdkafka
 
 # COMPOSER: install binary
 RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/bin --filename=composer
 
 # COMPOSER: install dependencies
 RUN composer global require hirak/prestissimo
 
-USER $HOST_USER
+USER 'www-data'
```

Comment:
please move this above the installation of prestissimo, it's an old bug that hasn't been fixed in all dockerfiles